From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Samuel Attard <sattard@slack-corp.com>
Date: Tue, 2 Apr 2019 14:38:43 -0700
Subject: fix: retain compatibility with MSVC++

The C++ standard has always forbidden containers of const elements such
as std::set<const T>.  In order to retain compability with MSVC++ this
code can not exist in Chromium.

This should probably also be fixed in libc++ to prevent this occurring
again but this fixes the build for now.

diff --git a/components/language/core/browser/language_prefs.cc b/components/language/core/browser/language_prefs.cc
index 317aa1d8298cd697c5388ddeb9680b39e562e9ab..df258ca62acfa8a198bcb7c4be31746a491d938b 100644
--- a/components/language/core/browser/language_prefs.cc
+++ b/components/language/core/browser/language_prefs.cc
@@ -85,7 +85,7 @@ void LanguagePrefs::ResetEmptyFluentLanguagesToDefault() {
 }
 
 base::Value LanguagePrefs::GetDefaultFluentLanguages() {
-  std::set<const std::string> languages;
+  std::set<std::string> languages;
 #if defined(OS_CHROMEOS)
   // Preferred languages.
   std::string language = language::kFallbackInputMethodLocale;
